docs/images.md

```diff
@@ -0,0 +1,41 @@
+# Images
+
+Rancher Desktop provides the ability to build, push, and pull images via the
+[KIM](https://github.com/rancher/kim) project.
```

Comment:
I think kim is all lowercase

Reply:
That's a good question. KIM is an acronym so it should be KIM. I just went and asked to double check.